tests/components/deconz/test_deconz_event.py

```diff
@@ -101,6 +109,20 @@ async def test_deconz_events(hass):
         "gesture": 0,
     }
 
+    gateway.api.sensors["5"].update(
+        {"state": {"buttonevent": 6002, "angle": 110, "xy": [0.5982, 0.3897]}}
+    )
+    await hass.async_block_till_done()
+
+    assert len(events) == 4
+    assert events[3].data == {
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)